descriptors/ModuleDescriptor-template.json

```diff
@@ -41,6 +41,44 @@
           "permissionsRequired": [ "search.instances.collection.get" ]
         }
       ]
+    },
+    {
+      "id": "search-config",
+      "version": "0.1",
+      "handlers": [
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/search/config/languages",
+          "permissionsRequired": [ "search.config.languages.item.post" ]
+        },
+        {
+          "methods": [
+            "GET"
+          ],
+          "pathPattern": "/search/config/languages",
+          "permissionsRequired": [ "search.config.languages.collection.get" ]
+        },
+        {
+          "methods": [
+            "DELETE"
+          ],
+          "pathPattern": "/search/config/languages/{id}",
+          "permissionsRequired": [ "search.config.languages.item.delete" ]
+        }
+      ]
+    },
+    {
+      "id" : "_tenant",
+      "version" : "2.0",
+      "interfaceType": "system",
+      "handlers": [
+        {
+          "methods": ["POST"],
+          "pathPattern": "/_/tenant"
+        }
+      ]
```

Comment:
I have created https://issues.folio.org/browse/MSEARCH-45 for this, we have to update folio-spring-base module so it should propagate x-okapi-tenant header.

I also downgraded tenant API version to 1.2 because we don't support the new async style.